.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Probably don't want to ignore `.idea`.

I prefer to keep it in `.global_gitignore`:

```
[core]
        excludesfile = ~/.gitignore_global
```

https://git-scm.com/docs/gitignore

Reply:
I got it, thanks
Done